README.md

```diff
@@ -581,6 +651,70 @@ With:
 -DENABLE_CURAND_BACKEND=True   \
 ```
 
+### Building with hipSYCL
+HipSYCL currently supports the BLAS domain with the following backends:
+* NETLIB
+* MKLCPU
+* CUBLAS
+* ROCBLAS
+
+From which the ROCBLAS backend is only available when compiling with hipSYCL
```

Comment:
Can we add a section under "Build Options" for hipSYCL - and add these information there?